alembic/env.py

```
@@ -21,18 +21,20 @@
 from src import engine
+
 target_metadata = engine.Base.metadata
-target_metadata = target_metadata 
+target_metadata = target_metadata
 
 # other values from the config, defined by the needs of env.py,
 # can be acquired:
 # my_important_option = config.get_main_option("my_important_option")
 # ... etc.
-if ( os.getenv("SQL_LITE_URL") == None):
+if os.getenv("SQL_LITE_URL") == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)